src/helpers/AllExternalPools.ts

```diff
@@ -70,6 +70,12 @@ export const allPools = [tj_gohm_wavax, sushi_arb_gohm_weth, sushi_poly_gohm_wet
  * @param address
  */
 export const fetchPoolData = async (address: string) => {
+  if (!address) {
+    // If we don't have the user address,
+    // maybe because the user is anonymous and is not connected to a wallet,
+    // then we can't calculate their liquidity pool ownership data.
+    return allPools;
+  }
```

Comment:
hey sorry, I just noticed this. This isn't correct. 
The previous code (below) is doing two things:
1. calculating a user's balance in a liquidity pool IF their address is present (see line 78/84) https://github.com/OlympusDAO/olympus-frontend/pull/1311/files#diff-b7abd1507ba55865993be284dadaab39b47cba5ac3af3d42845fefd2915c958fL78
2. calculating the pool's TVL, which needs to happen whether the user has a connected wallet or not.

The IF statement you added prevents the TVL from being calculated.

Reply:
here's a non-connected wallet on your PR... tvl doesn't load due to your IF statement
![image](https://user-images.githubusercontent.com/80423742/152644759-76c9d8b5-fd1a-4b8d-9b12-791e5be077c7.png)

Reply:
here's a non-connected wallet on staging, TVL is still calculated even if a wallet isn't connected 
![image](https://user-images.githubusercontent.com/80423742/152644793-f59104f8-7557-429c-86c8-2e457b0edb5e.png)